kratos/containers/distributed_csr_matrix.h

```diff
@@ -117,6 +117,7 @@ class DistributedCsrMatrix
 
         //*******************************
         //construct diagonal block
+<<<<<<< HEAD
```

Comment:
looks like there are merge conflicts to resolve

Reply:
yes i am on it.

I just merged the master